src/lib/permissions/PermissionsAPI.web.js

```diff
@@ -0,0 +1,30 @@
+class PermissionsAPIWeb {
+  async getCameraPermissionStatus() {
+    // Fetching the list of input/output media devices
+    const listOfMediaDevices = navigator.mediaDevices ? await navigator.mediaDevices.enumerateDevices : []
+
+    // Search for video input device (i.e. camera)
+    const videoDevice = listOfMediaDevices.find(device => device.kind === 'videoinput') || {}
+
+    // if label field is not empty or null - it means camera access is granted
+    // so returning label field as boolean value
+    return Boolean(videoDevice.label)
```

Comment:
it's just an camera existence check, have to check permissions also according to the algorithm i've posted in the comments

Reply:
done